mlops/bicep/open_sandbox_setup.bicep

```diff
@@ -45,7 +45,7 @@ param siloRegions array = [
 ]
 
 @description('The VM used for creating compute clusters in orchestrator and silos.')
-param compute1SKU string = 'Standard_DS3_v2'
+param compute1SKU string = 'Standard_NC6'
```

Comment:
shouldn't we keep CPU by default for compute1 but use GPU for compute2?

Reply:
As discussed, keeping the compute1 as-is and changing the compute2 SKU to `Standard_NC6`